enterprise/frontend/src/metabase-enterprise/transforms/components/JobView/ScheduleSection/ScheduleSection.tsx

```diff
@@ -122,3 +118,83 @@ function RunButtonSection({ job }: RunButtonSectionProps) {
     </Tooltip>
   );
 }
+
+type RunStatusSectionProps = {
+  job: TransformJobInfo;
+};
+
+function RunStatusSection({ job }: RunStatusSectionProps) {
+  const { last_run } = job;
+  const systemTimezone = useSetting("system-timezone");
+
+  if (last_run == null) {
+    return (
+      <Group gap="sm">
+        <Icon c="text-secondary" name="calendar" />
+        <Box>{t`This job hasn’t been run before.`}</Box>
+      </Group>
+    );
+  }
+
+  const { status, end_time, message } = last_run;
```

Comment:
We should make a common component and not duplicate it https://github.com/metabase/metabase/blob/c621b64c2fac7c604802242e996c40655c597daf/enterprise/frontend/src/metabase-enterprise/transforms/pages/TransformPage/RunSection/RunSection.tsx#L90. Perhaps `runsInfo` can be passed as a prop